subsys/bluetooth/shell/bt.c

```diff
@@ -622,6 +622,8 @@ static void connected(struct bt_conn *conn, uint8_t err)
 		}
 	}
 
+	default_conn = bt_conn_ref(conn);
```

Comment:
@Thalley, I think this line has been inserted by mistake as it does call twice `bt_conn_ref`. Is this intentional?

Reply:
@msmttchr https://github.com/zephyrproject-rtos/zephyr/pull/66484